storey/table.py

```diff
@@ -44,6 +46,20 @@ def __init__(self, table_path: str, storage: Driver, partitioned_by_key: bool =
     def __str__(self):
         return f'{self._container}/{self._table_path}'
 
+    def _get_lock(self, key):
+        key = get_hashed_key(key)
```

Comment:
no need to hash it, it's already hashed in all the places the call this function